eng/packages/http-client-csharp/ci.yml

```diff
@@ -10,58 +10,45 @@ pr:
     include:
     - eng/packages/http-client-csharp
 
+parameters:
+  - name: BuildPrereleaseVersion
```

Comment:
Is this regen triggered for each commit? To regen all the SDK in azure-sdk-for-net?

Reply:
Automatically?

Reply:
As configured, client regeneration would run unless we disable it a queue time. We can change the default for `ShouldRegenerate` to `false` if we only want to run cadl ranch tests for push triggered runs.  If so, we should disable publishing in the CI run also.  This would require a manual run to publish the emitter and regenerate the clients using the newly published version.